Sources/ConcurrencyHelpers/Protected.swift

```diff
@@ -0,0 +1,36 @@
+/// Thread safe access to simple variable protected by lock
+@propertyWrapper
+public final class Protected<T> {
```

Comment:
What kind of data supposed to be used with Protected<>?
swift-atomics package provides the same semantic for most plain generic types (like integers, bool, pointers)
Using more complex things like structs could be dangerous.

Reply:
I would like to have value type variable somewhere in my class and organise thread safe access to it. So in my first use cases:
1. AsyncStream continuation 
2. KafkaStatistics